stdlib/ballerina-http/src/main/ballerina/ballerina/net/http/resiliency/failover-connector.bal

```diff
@@ -0,0 +1,523 @@
+package ballerina.net.http.resiliency;
+
+import ballerina.net.http;
+
+@Description {value:"Represents a failover connector retry configuration."}
+@Field {value:"retryErrorCodes: Array of error codes which required retry the requests."}
+public struct FailOverRetryConfig {
+    int[] retryErrorCodes;
+    int interval;
+}
+
+@Description {value:"Failover connector struct represents an error occured during the Failover connector."}
```

Comment:
This should be `FailoverConnectorError` struct right?

Reply:
also, better to rephrase the last bit as `an error occurred in an action of the Failover connector`